index.html

```diff
@@ -103,6 +104,8 @@ <h3><a class="textdecoration font1" href="#">Tonic</a></h3>
                     <li><a class="textdecoration font3" href="#">Back End Dev</a></li>
                     <li class="textdecoration"><a href="#"><img src="./img/mid_dot.jpg" alt=""></a></li>
                     <li><a class="textdecoration font3" href="#">2015</a></li>
+                                    <!-- Image text alternativ should be added -->
+
```

Comment:
- [ ] Kindly make sure to the alternative text `alt` to describe your image as required.